tools/RAiDER/cli/raider.py

```diff
@@ -246,16 +250,11 @@ def calcDelays(iargs=None):
         wfiles = []
         for tt in times:
             try:
-                wfile = prepareWeatherModel(
-                        model, tt,
-                        ll_bounds=wm_bounds, # SNWE
-                        wmLoc=params['weather_model_directory'],
-                        makePlots=params['verbose'],
-                        )
+                wfile = prepareWeatherModel(model, tt, aoi.bounds(), makePlots=params['verbose'])
```

Comment:
we will check that the weather model bounds cover the aoi bounds and raise an exception if they dont